test/services/EventReportServiceSpec.scala

```diff
@@ -256,20 +256,29 @@ class EventReportServiceSpec extends AsyncWordSpec with Matchers with MockitoSug
   }
 
   "getEvent" must {
-    "return the payload from the connector when valid event type" in {
+    "return the payload from the connector when a valid event type is supplied for Api1832" in {
       when(mockEventReportConnector.getEvent(any(), any(), any(), any())(any(), any()))
         .thenReturn(Future.successful(responseJson))
       whenReady(eventReportService.getEvent(pstr, startDate, version, EventType.Event3)(implicitly, implicitly)) { result =>
         result mustBe responseJson
       }
     }
 
-    "return not found exception when invalid event type" in {
+    "return the payload from the connector when a valid event type is supplied for API1833" in {
+      when(mockEventReportConnector.getEvent(any(), any(), any(), any())(any(), any()))
+        .thenReturn(Future.successful(responseJson))
+      whenReady(eventReportService.getEvent(pstr, startDate, version, EventType.Event1)(implicitly, implicitly)) { result =>
+        result mustBe responseJson
+      }
+    }
+
+    //TODO: This test will be redundant once all the getEvent cases are implemented
```

Comment:
I think this will be valid test as long as we have following branch :
`case _ => Future.failed(new NotFoundException(s"Not Found: ApiType not found for eventType ($eventType)"))`

Reply:
I mean user can pass a valid eventTeype but the eventType might not be a GetApiType